core/frontend/src/IModelApp.ts

```diff
@@ -6,7 +6,10 @@
  * @module IModelApp
  */
 
-const copyrightNotice = 'Copyright © 2017-2022 <a href="https://www.bentley.com" target="_blank" rel="noopener noreferrer">Bentley Systems, Inc.</a>';
+/** @public */
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+export const ITWINJS_CORE_VERSION = require("../../package.json").version; // require resolves from the lib/{cjs,esm} dir
```

Comment:
@pmconne I know you mentioned exposing it as an object, but we already use [semver](https://www.npmjs.com/package/semver) so that should help.

Reply:
But surely it's at least a string, not `any`, right?

Reply:
correct; typed as a string now